src/content/docs/2/guide/upgrade-migrate/from-tauri-1.mdx

```diff
@@ -17,7 +17,7 @@ The Tauri v2 CLI includes a `migrate` command that automates most of the process
     npm run tauri migrate"
 	yarn="yarn upgrade @tauri-apps/cli --latest
     yarn tauri migrate"
-	pnpm="pnpm update @tauri-apps/cli --latest
+	pnpm="pnpm upgrade @tauri-apps/cli@next
```

Comment:
```suggestion
	pnpm="pnpm update @tauri-apps/cli@next
```

`pnpm upgrade` is an alias for `pnpm update` so I think it makes sense to keep this as-is, but good catch with `@next`!